### PR TITLE
Add statix checker

### PR DESCRIPTION
**Pull request by lunik1**
Adds the statix nix checker https://github.com/nerdypepper/statix

Requires a bit of trickery to deal with https://github.com/nerdypepper/statix/issues/59

### CODE REVIEW

**Comment by nerdypepper** [comment]
have released v0.5.8 which contains a fix for this, sorry for the slow movement with the project, life gets in the way!

**Comment by bbatsov** [comment]
@lunik1 Can you update the PR to account for the bugfix?

File: CHANGES.rst [hunk 1]
```diff
@@ -17,6 +17,10 @@
     which they were returned by checkers.  In previous versions of Flycheck,
     this list was sorted by error position and severity. [GH-1749]
 
+- New syntax checkers:
```
Comment (bbatsov) [inline CHANGES.rst:20]: Can you move this up the changelog? (flycheck 33 was just released)

**Comment by lunik1** [comment]
Done!

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**